### ISSUE TITLE
Add ROBOT template merge pipeline as option to ODK

### ISSUE THREAD

**Issue by matentzn** [INCATools/ontology-development-kit#579]
The purpose is to bulk add term data provided in the form of a table to an ontologies editors file, like in MONDO, RO and other ontologies.

**Comment by anitacaron** [comment]
Is this related to the option to support the ROBOT template in the components?

[Test example](https://github.com/INCATools/ontology-development-kit/blob/master/tests/test-templates.yaml)

If it's related to it, this was fixed in the commit https://github.com/INCATools/ontology-development-kit/commit/de2d837940c8830b42ab28bfe9e42d4af25c8233 and it has been available since ODK 1.4.

**Comment by matentzn** [comment]
No, it is adding this here as a .PHONY goal: https://github.com/monarch-initiative/mondo/blob/master/src/ontology/mondo.Makefile#L668 or https://github.com/information-artifact-ontology/ontology-metadata/blob/master/src/ontology/omo.Makefile#L13

This is super useful for cases where you want to rapidly push a template into the editors file, and not maintain it as a component.
